New docs content (#931)
liveblocks/liveblocks: docs/pages/concepts/why-liveblocks.mdx

---
meta:
  title: "Why Liveblocks"
  parentTitle: "Concepts"
  description:
    "Liveblocks is a real-time collaboration infrastructure for building
    performant collaborative experiences."
---

> Everything we do at Liveblocks comes from the strong belief that flexible office
> policies are here to stay, and as a result, every SaaS product will eventually
> be real-time collaborative to help people feel connected with one another and
> work better together.

Reviewer: ```suggestion
Everything we undertake at Liveblocks derives from the firm belief that flexible office
policies are here to stay, and that as a result, every SaaS product will eventually
be real-time collaborative, helping people feel more connected with one another and
work better together.
```

I've carefully written an alternative version too, in case you prefer it:

> At Liveblocks, we firmly believe that flexible office policies are here to stay, and that as 
> a result, all SaaS products will eventually use real-time collaboration. Every project
> we undertake at Liveblocks stems from this belief, and the belief that we can empower
> people to work better together, and feel more closely connected with one another.

Also an extra-confident alternative:

> At Liveblocks, we're certain that flexible office policies are here to stay. Every
> SaaS product will eventually need real-time collaboration, empowering more 
> effective work, and closer connections between people.

I really thought about these, I think the first paragraph is important.

Author: Love those, much much much better. Important to nail that paragraph since it's the first thing people will read.

## Why collaboration

> In the mid 2000s and after years of development, Google launched their
> browser-based real-time collaborative Microsoft Word competitor. This approach
> completely changed the way people worked together as it enabled them to
> collaborate efficiently in real-time but also asynchronously via comments. No
> more disjointed email threads where files would get passed around. As they were
> getting more markershares, Microsoft had no other choice but to build similar
> collaboration features right into Microsoft Word and the broader Microsoft
> Office suite in order to compete.

Reviewer: ```suggestion
In 2006, following years of development, Google launched their
browser-based real-time collaborative Microsoft Word competitor. This approach
transformed the way people work together enabling them to
collaborate efficiently in real-time but also asynchronously via comments—no
more disjointed email threads with files being passed around. As Google were
gaining market shares, Microsoft were forced to build similar
collaboration features directly into Microsoft Word and the broader Microsoft
Office ecosystem in order to compete.
```
Pick and choose!

<Figure highlight>
  <Image
    src="/images/docs/concepts/why-liveblocks-google.png"
    alt="Why Liveblocks - Google"
    layout="responsive"
    width={768}
    height={440}
    quality={90}
  />
</Figure>

> About a decade later, Figma flipped the whole software design industry upside
> down with their browser-based multiplayer screen design tool, and subsequently
> leaving leading companies in the space on the sideline. Very few designers
> believed a quality design tool could be built into the browser, and no designers
> really wanted to have their peers and managers looking over their shoulders
> while designing. Yet, they proved everybody wrong and a majority of designers
> started to use Figma. And because Figma was built with collaboration at its
> core, engineers, product managers, executives, marketers, copywriters quickly
> followed.

Reviewer: ```suggestion
Around a decade later, Figma flipped the whole software design industry on
its head with their browser-based tool for multiplayer design, and subsequently
left leading companies behind. Few designers
believed a quality design tool could be built into the browser, and no designers
wanted to have their peers and managers looking over their shoulders
while designing. Yet, they proved everybody wrong and a majority of designers
started to use Figma. And because Figma was built with collaboration at its
core, engineers, product managers, executives, marketers, and copywriters quickly
followed.
```
Style changes. 

I'm not sure who "they" and "everybody" is in this phrase: "they proved everybody wrong", I think this needs to be clearer

<Figure highlight>
  <Image
    src="/images/docs/concepts/why-liveblocks-figma.png"
    alt="Why Liveblocks - Figma"
    layout="responsive"
    width={768}
    height={440}
    quality={90}
  />
</Figure>

> Those companies had the technical expertise and millions of dollars to pull this
> off. A handful of forward-looking companies were also able to build incredible
> collaborative products and become leaders in their respective markets: Notion,
> Pitch, Canva, Miro, Mural, Linear, and a few others. While more and more
> companies are becoming aware of the importance of collaboration, only the ones
> that have gone through this process truly understand how painful and expensive
> this is to build and maintain.

Reviewer: ```suggestion
Those companies had the technical expertise, along with millions of dollars, to pull this
off. A handful of forward-looking companies were also able to build incredible
collaborative products and became leaders in their respective markets: Notion,
Pitch, Canva, Miro, Mural, Linear, to mention a few. While more and more
companies are becoming aware of the importance of collaboration, only the ones
that have gone through this process truly understand how painful and expensive
it is to build and maintain.
```
Style

<Figure highlight>
  <Image
    src="/images/docs/concepts/why-liveblocks-collaboration-pioneers.png"
    alt="Why Liveblocks - Figma"
    layout="responsive"
    width={768}
    height={440}
    quality={90}
  />
</Figure>

## Why choose Liveblocks

Liveblocks is a real-time collaboration infrastructure for building performant
collaborative experiences. We focus on the following five core pillars to enable

> companies to build awesome collaborative exeriences in a matter of days, not

Reviewer: ```suggestion
companies to build awesome collaborative experiences in a matter of days, not
```
Typo

months:

- A complete modular collaboration toolkit
- Deep integrations with your stack
- Pre-built components and examples
- Developer-centric tooling
- Fully-hosted solution

### A complete modular collaboration toolkit

> Liveblocks is a complete modular toolkit to embed collaborative experiences into

Reviewer: ```suggestion
Liveblocks is a complete modular toolkit for embedding collaborative experiences into
```

your product, enabling you to pick and choose the parts you need depending on

> the type of collaborative experiences you’re tyring to build and requirements.

Reviewer: ```suggestion
the type of collaborative experience you’re trying to build and its requirements.
```
Typo + clarity

### Deep integrations with your stack

Liveblocks integrates deeply with popular frontend frameworks and libraries,

> making it easy to embed real‑time collaborative experience into any product

Reviewer: ```suggestion
making it easy to embed real‑time collaborative experiences into any product
```
Typo

quickly.

### Pre-built components and examples

Liveblocks provides you with powerful open‑source examples and components that
can be used modularly, enabling you to move fast while focusing on your core
product.

### Developer-centric tooling

Liveblocks accelerates your productivity with a great developer experience

> throughout, DevTools, and analytics to understand how your users are using your

Reviewer: ```suggestion
throughout, including DevTools, and analytics to understand how your users are using your
```
